sql/moz-fx-data-experiments/monitoring/experimenter_experiments_v1/query.py

```diff
@@ -80,17 +87,20 @@ class ExperimentV1:
     variants: List[Variant]
     proposed_enrollment: Optional[int] = attr.ib(converter=_coerce_none_to_zero)
     firefox_channel: str
+    targeting: str
```

Comment:
```suggestion
```
I don't see `targeting` being defined for v1 experiments which get pulled in from this API: https://experimenter.services.mozilla.com/api/v1/experiments/